packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1241,7 +1243,32 @@ class CheckboxMenuButton extends StatelessWidget {
     this.trailingIcon,
     this.closeOnActivate = true,
     required this.child,
-  });
+  }) : _checkboxType = _CheckboxType.material;
+
+  /// Creates an adaptive [CheckboxMenuButton].
+  ///
+  /// The checkbox uses [Checkbox.adaptive] to show a [CupertinoCheckbox] for
+  /// iOS platforms, or [Checkbox] for all others.
```

Comment:
Nit: I would use the description in Checkbox.adaptive (but tweaked maybe) - https://github.com/flutter/flutter/blob/3ed3f31ba69729809ee5240397e487513d726c56/packages/flutter/lib/src/material/checkbox.dart#L104-L107